CHANGELOG.md

```diff
@@ -4,12 +4,33 @@ The format is based on [Keep a Changelog].
 
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
+## [v0.19.0] - 2023-07-20
+
+### [Fixed]
+
+- Fixed connections processing await on server shutdown  ([#1153](https://github.com/paritytech/jsonrpsee/pull/1153))
+- fix: include error code in RpcLogger  ([#1135](https://github.com/paritytech/jsonrpsee/pull/1135))
+- fix: downgrade more logs to `debug`  ([#1127](https://github.com/paritytech/jsonrpsee/pull/1127))
+- fix(server): remove `MethodSinkPermit` to fix backpressure issue on concurrent subscriptions  ([#1126](https://github.com/paritytech/jsonrpsee/pull/1126))
+- fix readme links  ([#1152](https://github.com/paritytech/jsonrpsee/pull/1152))
+
+### [Changed]
+
+- server: downgrade connection logs to debug  ([#1123](https://github.com/paritytech/jsonrpsee/pull/1123))
+- refactor(server): make `Server::start` infallible and add `fn builder()`  ([#1137](https://github.com/paritytech/jsonrpsee/pull/1137))
+
+### [Added]
+
+- added uptest new rust lib  ([#1151](https://github.com/paritytech/jsonrpsee/pull/1151))
+- Add subway  ([#1124](https://github.com/paritytech/jsonrpsee/pull/1124))
```

Comment:
Nit: this isn't really important (and should prolly be "Add Subway to users list"

Reply:
Yep that makes sense, will remove this since I'd place it with the same priority as `chore` PRs 👍